core/domain/state_domain.py

```diff
@@ -841,6 +842,370 @@ def get_all_outcomes(self):
             outcomes.append(self.default_outcome)
         return outcomes
 
+    def validate_numeric_input(self):
```

Comment:
For private functions that are only called within the class, it's good style for readability to use an underscore like `def _validate_numeric_input(self)`

Reply:
Done